Problem running scenario 2 on Ubuntu with snap-based Node.js

When trying `make scenario2-setup`, i get:

```
rm -r ~/.ag-chain-cosmos
rm: impossible de supprimer '/home/david/.ag-chain-cosmos': Aucun fichier ou dossier de ce type
make: [Makefile:42: scenario2-setup] Error 1 (ignorée)
rm ag-cosmos-chain-state.json
rm: impossible de supprimer 'ag-cosmos-chain-state.json': Aucun fichier ou dossier de ce type
make: [Makefile:43: scenario2-setup] Error 1 (ignorée)
./lib/ag-chain-cosmos init --chain-id=agoric
Error: /snap/core/current/usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.22' not found (required by /home/david/projects/cosmic-swingset/build/Release/agcosmosdaemon.node)
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:977:18)
make: *** [Makefile:44: scenario2-setup] Error 1
```

The problems with the `rm`s are expected i imagine. However, it's less clear to me the other error (about `GLIBCXX_3.4.22`) is expected

--- a/README.md
+++ b/README.md
@@ -24,6 +24,7 @@ In any case, for now, you will be needing to build the solo node from the source
 If you want to build and install from sources, you need to install
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [node.js](https://nodejs.org/en/) (you need at least version 11). This installs 2 binaries named `node` (JavaScript runtime) and `npm` (JavaScript package manager)
+    - **Warning:** There are some [known issues](https://github.com/Agoric/cosmic-swingset/issues/71) installing cosmic-swingset with [snap-based version of node.js on Ubuntu](https://github.com/nodesource/distributions/blob/master/README.md#snap). **We recommand** using a non-snap version
 - [Golang](https://golang.org/doc/install) (you need at least version 1.12)
 
 ```sh

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ In any case, for now, you will be needing to build the solo node from the source
 If you want to build and install from sources, you need to install
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [node.js](https://nodejs.org/en/) (you need at least version 11). This installs 2 binaries named `node` (JavaScript runtime) and `npm` (JavaScript package manager)
-    - **Warning:** There are some [known issues](https://github.com/Agoric/cosmic-swingset/issues/71) installing cosmic-swingset with [snap-based version of node.js on Ubuntu](https://github.com/nodesource/distributions/blob/master/README.md#snap). **We recommand** using a non-snap version
+    - **Warning:** There are some [known issues](https://github.com/Agoric/cosmic-swingset/issues/71) installing cosmic-swingset with [snap-based version of node.js on Ubuntu](https://github.com/nodesource/distributions/blob/master/README.md#snap). **We recommend** using a non-snap version
 - [Golang](https://golang.org/doc/install) (you need at least version 1.12)
     - **(optional)** If installing the GO language didn't setup a `$GOPATH` variable, you'll need to find the directory and set the variable. Typically `GOPATH="$HOME/go"`
 

--- a/README.md
+++ b/README.md
@@ -26,6 +26,7 @@ If you want to build and install from sources, you need to install
 - [node.js](https://nodejs.org/en/) (you need at least version 11). This installs 2 binaries named `node` (JavaScript runtime) and `npm` (JavaScript package manager)
     - **Warning:** There are some [known issues](https://github.com/Agoric/cosmic-swingset/issues/71) installing cosmic-swingset with [snap-based version of node.js on Ubuntu](https://github.com/nodesource/distributions/blob/master/README.md#snap). **We recommand** using a non-snap version
 - [Golang](https://golang.org/doc/install) (you need at least version 1.12)
+    - **(optional)** If installing the GO language didn't setup a `$GOPATH` variable, you'll need to find the directory and set the variable. Typically `GOPATH="$HOME/go"`
 
 ```sh
 git clone git@github.com:Agoric/cosmic-swingset.git
@@ -38,16 +39,6 @@ Make symbolic links somewhere in your `$PATH` (such as `/usr/local/bin`) as belo
 
 ```sh
 ln -s $PWD/lib/ag-chain-cosmos /usr/local/bin/
-```
-
-If installing the GO language didn't setup a `$GOPATH` variable,
-you'll need to find the directory and set the variable. Typically
-```sh
-GOPATH="$HOME/go"
-```
-
-Then do
-```sh
 ln -s $GOPATH/bin/ag-cosmos-helper /usr/local/bin/
 ```
 

